src/components/Transfer/components/ShownTransfer/components/List/components/Item/hooks/useDeleteOption.ts

```diff
@@ -1,10 +1,8 @@
 import { useCallback } from 'react';
-import { OptionType } from '../../../../../../../../Select';
+import { UseTransferOptionT } from '../../../../../../..';
```

Comment:
tohle je circular dependency, v tomhle repu na to chybí check (zatím), je potřeba použít import, kde je ten typ definovaný